utils-testing/src/main/java/pm.gnosis.tests.utils/Asserts.kt

```diff
@@ -3,7 +3,7 @@ package pm.gnosis.tests.utils
 import org.junit.Assert
 
 object Asserts {
-    fun assertThrow(test: () -> Unit, message: String? = null, throwablePredicate: ((Throwable) -> Boolean)? = null) {
+    inline fun assertThrow(test: () -> Unit, message: String? = null, noinline throwablePredicate: ((Throwable) -> Boolean)? = null) {
```

Comment:
Is this change necessary? I am not sure that there is a clear benefit of making this function `inline`

https://kotlinlang.org/docs/reference/inline-functions.html#noinline